locales/it/translation.json

```
@@ -0,0 +1,250 @@
+        "fixed": "Fisso",
+        "flexible": "Flessibile",
+        "language": "Linguaggio"
+    },
+    "$Menu": {
+        "about": "Informazioni",
+        "all-clear": "Tutto pulito!",
+        "all-files": "Tutti i files",
+        "cancel": "Annulla",
+        "check-for-updates": "Cerca aggiornamenti",
+        "clear-database": "Pulisci database",
+        "confirm-clear-all-data": "Sei sicuro di voler pulire tutti i dati?",
+        "confirm-import-db": "Sei sicuro di voler importare un database? Sovrascriverà tutte le informazioni attualmente presenti.",
+        "confirm-migrate-to-flexible": "Sei sicuro di voler migrare il database con calendario fisso sul calendario flessibile?\n\nIl database con calendario flessibile sarà cancellato.",
+        "copy": "Copia",
+        "could-not-be-loaded": "non può essere caricato",
+        "cut": "Taglia",
+        "database-export": "Esporta database",
+        "database-imported": "Database importato",
+        "database-migrated": "Database migrato",
+        "database-was-exported": "Okay, il database è stato esportato.",
+        "edit": "Modifica",
+        "exit": "Esci",
+        "export-database": "Esporta database",
+        "export-db-to-file": "Esporta database su file",
+        "export": "Esporta",
+        "failed-entries": "Record falliti",
+        "failed-migrating": "Fallimento migrando",
+        "help": "Aiuto",
+        "import-database": "Importa database",
+        "import-db-from-file": "Importa database da file",
+        "import-successful": "Yay! Importazione conclusa con successo!",
+        "import": "Importa",
+        "menu": "Menu",
+        "migrate-calendar-to-flexible": "Migra database con calendario fisso a flessibile",
+        "migrate-to-flexible-calendar": "Migrate a calendar flessibile",
+        "migration-successful": "Yay! Migrazione conclusa con successo!",
+        "no-thanks": "No, grazie",
+        "ok": "OK",
+        "paste": "Incolla",
+        "preferences": "Preferenze",
+        "punch-time": "Ora di registrazione",
+        "punched-time": "Orario di registrazione avvenuta",
+        "quit": "Esci",
+        "reload": "Ricarica",
+        "select-all": "Seleziona tutto",
+        "send-feedback": "Manda feedback",
+        "should-migrate-to-flexible": "Il tuo calendario flessibile è vuoto. Vuoi cominciare migrando il database con calendario fisso sul calendario flessibile?",
+        "show-app": "Mostra App",
+        "something-went-wrong": "Qualcosa è andato storto :(",
+        "toggle-dev-tools": "Abilita Developer Tools",
+        "ttl-github": "TTL GitHub",
+        "view": "Mostra",
+        "workday-waiver-manager": "Gestore giorni lavorativi esclusi",
+        "yes-please": "Sì, per favore"
+    },
+    "$FlexibleDayCalendar": {
+        "switch-view": "Cambia visuale",
+        "previous-day": "Giorno precedente",
+        "next-day": "Giorno successivo",
+        "time-to-leave": "Ora di andare",
```

Time to Leave is the name of the app, because of this we have not been translating it:

```suggestion
        "time-to-leave": "Time to Leave",
```